bundles/org.openhab.binding.omnilink/src/main/resources/OH-INF/i18n/omnilink.properties

```diff
@@ -318,3 +318,10 @@ channel-type.omnilink.zone_latched_alarm_status.state.option.1 = Tripped
 channel-type.omnilink.zone_latched_alarm_status.state.option.2 = Reset, but previously tripped
 channel-type.omnilink.zone_restore.label = Restore Zone
 channel-type.omnilink.zone_restore.description = Send a 4 digit user code to restore this zone.
+
+# thing actions
+
+actionInputZoneLabel = Time zone
+actionInputZoneDesc = The time zone of the controller, provided in the "America/Denver" format.
+actionLabel = Set the Date/Time of the controller
+actionDesc = Sets the current Date/Time of the controller to the correct value, including daylight savings.
```

Comment:
The "correct value" is in this case the date and time of the system where openHAB is running. Maybe this could be written more precisely to avoid any doubts? Perhaps something like: "Synchronizes the system time and DST of the controller with openHAB's system time."

Reply:
Well, that is not necessarily true, because it synchronizes it with the provided locale in the action input, right?